src/main/java/org/example/final02/Final02Application.java

```diff
@@ -8,6 +8,7 @@ public class Final02Application {
 
     public static void main(String[] args) {
         SpringApplication.run(Final02Application.class, args);
+        System.out.println("Hello World");
```

Comment:
Do we need this?

Reply:
no, this was only used to correct the commit